src/persistence.rs

```diff
@@ -1,23 +1,23 @@
-use std::str;
 use std::error::Error;
+use std::str;
 
-use std::io::{Read};
 use byteorder::{LittleEndian, ReadBytesExt, WriteBytesExt};
+use std::fs;
 use std::fs::File;
 use std::io;
-use std::fs;
+use std::io::Read;
 
 use crate::model_instance;
+use crate::optimizer;
 use crate::regressor;
 use crate::vwmap;
-use crate::optimizer;
+use clap::{App, AppSettings, Arg};
```

Comment:
We're trying to keep separation of functionalities in logical units.
Could you please move everything regarding command line parameter processing to model_instance.rs

I would suggest creating a function that is analogous to new_from_cmdline() ... for example update_from_cmdline()

Reply:
Thanks, refactored the replacement logic to `update_hyperparameters_from_cmd` in `model_instance.rs` - the fingerprint in `persistence.rs` is now minimal (one optional call to the update method).